debezium-core/src/main/java/io/debezium/connector/common/BaseSourceTask.java

```diff
@@ -87,6 +90,9 @@ public final void start(Map<String, String> props) {
 
             this.props = props;
             Configuration config = Configuration.from(props);
+            retriableRestartWait = config.getInteger(CommonConnectorConfig.RETRIABLE_RESTART_WAIT);
```

Comment:
Can you expose a dedicated accessor on `CommonConnectorConfig`, please?

Reply:
Yep, I'll add an accessor, but it will not be accessible here. Since we don't have an instance of CommonConnectorConfig.